services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/allu/AlluCommonDomain.kt

```diff
@@ -20,12 +20,13 @@ data class Contact(
 data class Customer(
     val type: CustomerType,
     val name: String,
-    val country: String, // ISO 3166-1 alpha-2 country code
+    val postalAddress: PostalAddress?,
```

Comment:
Looks like the postal address is only relevant for the invoicing customer (based on Figma). Maybe we could have a separate type for that, so the Customer class doesn't have fields that are always null. Postal address, ovt, invoicingOperator and sapCustomerNumber could all be in that class but not here.

Or maybe do this change to `application.Customer`, but not here.

Reply:
Postal address is part of the Allu API for Customer. That's why I added it here.